NuRadioMC/EvtGen/NuRadioProposal.py

```diff
@@ -534,7 +534,7 @@ def __filter_secondaries(self,
                 distance += ((sec.position.z - lepton_position[2]) * units.cm) ** 2
                 distance = np.sqrt(distance)
 
-                energy = self.__secondary_energy(sec)
+                energy = self.__secondary_energy(sec) * units.MeV
```

Comment:
Is this conversion done so that the output shower list has the correct NuRadioMC units? 
Did you verify that this `energy` is not used anywhere else within the proposal code (where it would have the wrong units)?

Reply:
Yes, I checked both